DemocracyClub/WhoCanIVoteFor
# Move results to postelection

Closes https://github.com/DemocracyClub/WhoCanIVoteFor/issues/1719

To test: 

- Run the ballot importer
- Run the server and check that the results for both the post elections related to `local.haringey.2023-10-04/haringey-local-election/` are unique.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205782782568392
